tests/paramgrill.c

```diff
@@ -1589,33 +1586,33 @@ static int allBench(BMK_result_t* resultPtr,
         return WORSE_RESULT;
     }
 
-    CBENCHMARK(loopDurationC < TIMELOOP_NANOSEC, benchres, tmp, BMK_compressOnly, BMK_timeMode, 1);
-    CBENCHMARK(loopDurationD < TIMELOOP_NANOSEC, benchres, tmp, BMK_decodeOnly,   BMK_timeMode, 1);
+    CBENCHMARK(loopDurationC < TIMELOOP_NANOSEC, benchres, tmp, BMK_compressOnly, 1);
```

Comment:
Do we need both of these runs? Line 1572 and line 1589.

Reply:
Looking at these lines, I suspect these tests are no longer active,
because `loopDurationC` and `loopDurationD` should no longer be `< TIMELOOP_NANOSEC / 10`,
since all tests use `BMK_timeMode` now.

I could add an assert() to ensure it, and then remove these lines.